pkgs/by-name/to/tokyonight-gtk-theme/package.nix

```
@@ -6,7 +6,11 @@
   sassc,
   gnome-themes-extra,
   gtk-engine-murrine,
-  colorVariants ? [] # default: install all icons
+  colorVariants ? [ ],
+  sizeVariants ? [ ],
+  themeVariants ? [ ],
+  tweakVersions ? [ ],
```

```suggestion
  tweakVariants ? [ ],
```

for sake of simplicity, i would change this to also use `Variants`